internal/usage/usage.go

```diff
@@ -428,4 +428,6 @@ dbName and collection are required only for built-in roles.`
 	CompletedAfter                            = "Date filter of which the backup snapshots where completed. Format must be YYYY-MM-DD"
 	Tag                                       = "List that contains key-value pairs between 1 to 255 characters in length for tagging and categorizing the cluster."
 	UpdateWarning                             = " Passing this flag replaces preexisting data."
+	EnableWatch                               = "Watch the command until it completes its execution."
+	WatchTimeout                              = "Time in seconds after a watch command times out."
```

Comment:
The way this is worded sounds like it displays the time that has elapsed since the watch command timed out. If I'm not mistaken, what was meant was the following:
```suggestion
	WatchTimeout                              = "Time in seconds until a watch times out. After a watch times out, the CLI no longer watches the command."
```